### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by WouterSteen**

### CODE REVIEW

**Comment by Caroga** [comment]
I would also put a reference in the "how to get started section" 
`Make sure that you have the prerequisites installed and running correctly before proceeding. If you are looking to _update your setup than look here_.`

File: README.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ We follow a [Code of Conduct](docs/code-of-conduct.md).
 		* [By using DNSMASQ.](#by-using-dnsmaws)
 		* [Or by using the hostfile.](#or-by-using-the-hostfile)
 	* [3). Now, setup your projects!](#3-now-setup-your-projects)
+* [Update from exsisting environment.](#update)
```
Comment (Caroga) [inline README.md]: Fix typo: existing.
Also: `Update your existing installation` would be a better fit.

File: README.md [hunk 2]
```diff
@@ -145,6 +146,16 @@ It will try to connect to the host `172.17.0.1:9000` by default.
 Make sure to add a file mapping in your IDE:
 `./workspace/customer/project` => `/data/customer/project`
 
+# Update from exsisting environment
+
+If you update from a existing environment, follow these steps:
```
Comment (Caroga) [inline README.md]: `To update your existing installation, follow these steps`

File: README.md [hunk 3]
```diff
@@ -145,6 +146,16 @@ It will try to connect to the host `172.17.0.1:9000` by default.
...
+- dev down
```
Comment (Caroga) [inline README.md]: All commands should be encapsulated with ` to identify them as commands, not just as text.

**Comment by JeroenBoersma** [comment]
For now I've processed the typos and suggestions to the README, will merge it and we can move it to another section later.

**Comment by Caroga** [comment]
:+1:

**Pull request closed without merging**